src/main/java/in/org/projecteka/hdaf/link/link/LinkAccountController.java

```diff
@@ -0,0 +1,23 @@
+package in.org.projecteka.hdaf.link.link;
+
+import in.org.projecteka.hdaf.link.link.model.PatientLinkReferenceRequest;
+import in.org.projecteka.hdaf.link.link.model.PatientLinkReferenceResponse;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RestController;
+import reactor.core.publisher.Flux;
+
+@RestController
+@AllArgsConstructor
+public class LinkAccountController {
+
+    private LinkAccounts linkAccounts;
+
+    @PostMapping("/patients/link")
+    public Flux<PatientLinkReferenceResponse> linkAccounts(@RequestHeader(value="Authorization") String authorization,  @RequestBody PatientLinkReferenceRequest patientLinkReferenceRequest) {
+        return linkAccounts.linkAccounts(authorization, patientLinkReferenceRequest);
```

Comment:
linkAccounts.linkAccounts doesn't sound right. Can it be renamed to AccountService.Link ?